dozer-ingestion/src/errors.rs

```diff
@@ -123,6 +123,9 @@ pub enum ConfigurationError {
 
 #[derive(Error, Debug)]
 pub enum PostgresConnectorError {
+    #[error("Invalid SslMode: {0}")]
+    InvalidSslError(String),
```

Comment:
Let's put `SslMode` inside instead of a `String`.

Reply:
Sure

Reply:
We can do `#[error("Invalid SslMode: {0:?}")]`. That's what the current error message is anyway.

Reply:
Oh! didn't know we can do that haha will change it that way